src/plugins/info-factory.js

```diff
@@ -20,6 +21,7 @@ const infos = [
   AudioPlugin,
   VideoPlugin,
   ImagePlugin,
+  PdfViewerPlugin,
```

Comment:
Honestly, I did not think too much about it but subconsciously I probably did not want to be so bold and create "THE" PDF plugin so there is still room for other plugins dealing with PDFs (e.g. a document list for downloads) etc. I'm more OK with the other generic names because they actually implement the native HTML counterpart. But if you like the pure "PDF" better, I'd be glad to change it.

Reply:
Got it, sure.